src/tests/ftest/util/server_utils.py

```diff
@@ -127,14 +124,12 @@ def create_server_yaml(basepath, log_filename):
     # Write default_value_set dictionary in to AVOCADO_FILE
     # This will be used to start with daos_server -o option.
     try:
-        with open('{}/{}'.format(build_vars["PREFIX"],
-                                 AVOCADO_FILE), 'w') as write_file:
+        with open(avocado_yaml, 'w') as write_file:
```

Comment:
IIUC, this file will be under `$PREFIX/lib/daos/...`, is that correct?  If so, this file is not going to be writable in an RPM based environment.  It should be in `$PREFIX/etc/daos/` probably.